docs/coding-style.md

```diff
@@ -27,7 +27,17 @@ Static fields and properties should be placed before instance ones.
 
 Static methods are preferred to be after instance methods.
 
-Individual members must be separated by empty line, except sequence of constants, fields and abstract members. These should not be separated by empty lines.
+Methods which are called by other methods in the same category should be placed below the callers.
+
+```csharp
+public int Bar() => 2;
+
+int CallerOne(int x) => Foo() + x;
+
+int CallerTwo(int x) => Foo() + x + Bar();
```

Comment:
I find it difficult to assign a semantic meaning to `Foo` and `Bar` and keep track of that while reading "foo plus x plus bar".
I think `Foo` and `Bar` should be avoided unless the names have no semantic meaning and no effect at all, like in some test cases.

I propose to rename it like this:
```
int CallerTwo(int x) => Leaf() + x + Other();
```
or like this, in case "Bar" was meant to highlight the fact that it's public.
```
int CallerTwo(int x) => Leaf() + x + PublicMethod();
```

Reply:
fair point, I've simplified also to remove the `x`